podcasts/ABTest/ABTestProvider.swift

```diff
@@ -0,0 +1,27 @@
+import AutomatticTracks
+
+struct ABTestProvider: ABTestProviding {
+    static let shared = ABTestProvider()
+
+    func variation(for abTest: ABTest) -> Variation {
+        ExPlat.shared.experiment(abTest.rawValue) ?? .control
+    }
+
+    func start() async {
+        await withCheckedContinuation { continuation in
+            let experiments = ABTest.allCases.map { $0.rawValue }
+            if experiments.isEmpty {
+                return continuation.resume()
+            }
+
+            ExPlat.shared.register(experiments: experiments)
+            ExPlat.shared.refresh {
```

Comment:
It seems that assignments are batched all at once, am I understanding that correctly? This requires A/B tests to use exposure events to reduce data collection noise.

Is there an option to call for the assignment of a single experiment?